integration-tests/build.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
+    id 'groovy'
```

Comment:
Why you add groovy plugin - I don't see any .groovy files to run. This won't affect on building files it will use anyway installed groovy and gradle version on machine that will execute code

Reply:
I added it because of the possibility of working with json in separate Groovy files.